zivtech/vagrant-development-vm
# Use mailhog rather than custom mail handler.

Mailhog and the associated mhsendmail can are used to catch mail
sent from the machine and allow the messages to be viewed in the
browser in a nice UI. Here we replace sendmail with mhsendmail
and ensure mailhog is running via systemd.

- Remove the older mail override method
- Update readme
- Adjust puppet dependencies so we don't rely on multiple PHP
  versions.

## To test
- run `vagrant provision` on the vm and ensure there are no errors.
- Use a site that sends mail to send a message.
- visit mailhog.local.zivtech.com:8025 (actually *.local.zivtech.com:8025 will work)
- Ensure you see the email message arrive.
